src/generateDapp.ts

```diff
@@ -138,6 +138,9 @@ export async function generateDapp(selection: Selections) {
       case "boilerplate-template":
         await generateEnvFile();
         break;
+      case "nextjs-boilerplate-template":
+        await generateEnvFile();
```

Comment:
I think you have lost this change after rebase https://github.com/aptos-labs/create-aptos-dapp/commit/d5deef74e2092e62d31f94ca688ea8dcf8f307f8#diff-0594d46782df78830c5efe0fa4abb36cbf22c918dc182ccc8f3cf75fdee908e7R85

Reply:
should be fixed